recipes/czifile/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "czifile" %}
+{% set version = "2018.10.18" %}
+{% set sha256 = "ec5078897170fea3d267f6e00c64e200ec90519ea46973732235e2d958daabd8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - scipy
+    - tifffile=2018.10.18
+    - imagecodecs=2018.10.22
```

Comment:
This should be `imagecodecs ==2018.10.22`. Same for all other version comparison. `conda-build` usually expects there to be a space before all comparisons.